lib/ood_core/job/info.rb

```diff
@@ -143,6 +155,16 @@ def eql?(other)
       def hash
         [self.class, to_h].hash
       end
+
+      private
+
+      # Generate an aggregate status from child tasks
+      # @return [OodCore::Job::Status]
+      def job_array_aggregate_status
+        @tasks
+         .map { |task_status| task_status.status}
+         .max
+      end
```

Comment:
This is an example where since a single line is < 80 characters, I actually find the single line statement easier to read. Maybe its because I'm just used to seeing it that way?

```ruby
def job_array_aggregate_status
  @tasks.map { |task_status| task_status.status}.max
end
```

What style guide are you using?

Reply:
I am not currently using a style guide. Most of my interest in one at the beginning turned out to be trying to understand all the things that can be done with Ruby.

Reply:
I thought about it more and I can see the value in representing a chain vertically, especially when it gets longer. Also, the “complexity at a glance” might not be the best metric to grade code readability on - which is what I used in my initial comment.